client/src/components/Related/Related.jsx

```diff
@@ -8,9 +8,9 @@ const Related = (props) => {
   const [relatedProducts, setRelatedProducts] = useState();
```

Comment:
This is the top-most component of my module - it is basically a container for my RelatedProducts and YourOutfit components, and passes down necessary data as props.